test/bazel/proto_with_deps_test.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {DeliveryPerson} from "ts_protoc_gen/test/bazel/proto/common/delivery_person_pb";
```

Comment:
Maybe a silly question, but why use jasmine here - it looks to me like it's adding additional complexity - would it not be simpler to use mocha like the other tests do? This may negate the need for AMD-style modules.

Reply:
Unfortunately, the bazel typescript rules only support jasmine -- there's a [feature request](https://github.com/bazelbuild/rules_typescript/issues/124) for supporting other frameworks.

Reply:
If you feel experimental there are two open PRs to add mocha support: https://github.com/bazelbuild/rules_nodejs/pull/383

Reply:
Thanks for pointing those out, I hadn't seen them. However, I'm hesitant to depend on the mocha rules before they're merged into the main repo.